src/styles/main-file.scss

```diff
@@ -0,0 +1,21 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap");
+
+@import "./utility/variebles";
+@import "./utility/mixin";
+
+@import "./blocks/header.scss";
+@import "./blocks/main.scss";
+@import "./blocks/section-title.scss";
+@import "./blocks/recommended.scss";
+@import "./blocks/product.scss";
+@import "./blocks/categories.scss";
+@import "./blocks/how-to-buy.scss";
+@import "./blocks/button.scss";
+@import "./blocks/contacts.scss";
+@import "./blocks/form.scss";
+@import "./blocks/footer.scss";
+
+body {
```

Comment:
It's not a good practice to use global styles in this file. Consider moving these global styles to a separate file like '_base.scss' or '_reset.scss' and then importing it.